src/Compilers/CSharp/Portable/Symbols/LocalSymbol.cs

```diff
@@ -268,7 +268,7 @@ internal virtual bool IsWritable
                     case LocalDeclarationKind.UsingVariable:
                         return false;
                     default:
-                        return true;
+                        return RefKind != RefKind.RefReadOnly;
```

Comment:
The local is still assignable. Is that not what is being tracked here?

Reply:
So what do you expect to happen with the following code?

```csharp
int x = 0;
ref readonly int xr = ref x;
xr++;
```